packages/app/src/specs/SpecsListRowItem.vue

```diff
@@ -20,28 +20,54 @@
       >
         <div
           data-cy="specs-list-row-git-info"
-          class="col-span-2"
         >
           <slot name="git-info" />
         </div>
-        <div>
-          <slot name="latest-runs" />
-        </div>
-        <div class="hidden md:col-span-2 md:block">
-          <slot name="average-duration" />
-        </div>
+        <SpecsListHoverCell
+          data-cy="specs-list-row-latest-runs"
+          :is-hover-disabled="hasRuns"
```

Comment:
Nice, I like this `is` convention for booleans.

Reply:
👍